coderd/database/migrations/000227_notifications_workspace_autoupdated.up.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO notification_templates (id, name, title_template, body_template, "group", actions)
+VALUES ('c34a0c09-0704-4cac-bd1c-0c0146811c2b', 'Workspace updated automatically', E'Workspace "{{.Labels.name}}" updated automatically',
+        E'Hi {{.UserName}}\n\Your workspace **{{.Labels.name}}** has been updated automatically to the latest template version ({{.Labels.template_version_name}}).',
```

Comment:
Perhaps we should explain that this would have restarted it, as well?

Reply:
Oh, I think it wasn't restarted due to the update, but rather updated on the autostart. There is a configuration option to force the latest template version.

Reply:
How can we update a template without restarting a workspace?

Reply:
My assumption is as follows:

when the workspace is stopped by user or auto-stopped, and the next morning Lifecycle Executor brings it back to life, it will force the latest template version. I don't think we have a feature that stops the workspace and restarts with the latest template version?

Reply:
Oh I see what you mean! Gotcha.